examples/examples/models.py

```diff
@@ -1,4 +1,5 @@
 from django.db import models
+from django.utils.translation import gettext as _
```

Comment:
Is this left intentionally as a `_`?

Reply:
Yes. Because now we're in an app, and we should do it the normal ideomatic django way.. I think :P